sorc/ncep_post.fd/INITPOST_NETCDF.f

```diff
@@ -2978,6 +3003,27 @@ SUBROUTINE INITPOST_NETCDF(ncid2d,ncid3d)
      if(debugprint)print*,'sample stc = ',1,stc(isa,jsa,9)
 
       END IF
+!
+! E. James - 27 Sep 2022: this is for RRFS, adding smoke and dust
+! extinction; it needs to be after ZINT is defined.
+!
+      if (modelname == 'FV3R') then
+       do l = 1, lm
+        do j = jsta_2l, jend_2u
+         do i = ista_2l, iend_2u
+            taod5503d ( i, j, l) = extsmoke ( i, j, l ) + extdust ( i, j, l )
```

Comment:
@EricJames-NOAA If variables extsmoke and extdust are not used other in  routines, you might locally define, allocate and deallocate them in this routine for saving memory usage.

Reply:
Good idea...I have implemented that.  Let me know if you think any of the allocation / deallocation should be moved elsewhere within INITPOST_NETCDF.f